sdk/resourcemanager/README.md

```diff
@@ -96,6 +96,36 @@ See [Authentication][authenticate] for more options.
 
 See [Samples][sample] for code snippets and samples.
 
+### Standalone packages
```

Comment:
In the 
https://github.com/Azure/azure-sdk-for-java/blob/e1bc7c8ddb8542a0f0bbc17053b5367217a11dc7/sdk/resourcemanager/README.md#include-the-package

"Include the package" section, add a list of all services that are included. and say something like this

"For your convenience, we have provided a wrapper package that includes some of the most highly used Azure services. We recommend using this package you are dealing with multiple services that are included in this list"

Reply:
Yes, I think you also need to deal with the section `### Include single package` together. (merge or replace or anything else)